test/test.js

```diff
@@ -156,9 +157,50 @@ app.post('/api/v3/campaigns/32a381c49a2df99f1d0c6f3c112352b9.json', function (re
     sendFixture(res, 'create_campaign');
 });
 
+//---------------------------------------------------------------------------
+// Transactional requests
+//---------------------------------------------------------------------------
+app.get('/api/v3.1/transactional/smartemail/', function (req, res) {
```

Comment:
This should be:

    app.get('/api/v3.1/transactional/smartemail?', function (req, res) {